src/Http/SessionAuthenticator.php

```diff
@@ -22,6 +22,8 @@ public function logIn(Session $session, $token)
         // Backwards compatibility with $userId as parameter
         // Remove in beta 17
         if (! ($token instanceof AccessToken)) {
+            trigger_error('Parameter $userId is deprecated in beta 16, will be replaced by $token in beta 17', E_USER_DEPRECATED);
+
             $token = AccessToken::generate($token);
```

Comment:
Shouldn't this be a SessionAccessToken?

Reply:
I figured since the BC is there, I might as well leave it untouched. It will be removed in next release